docs/build-on-linea/volume-creation.mdx

```diff
@@ -0,0 +1,88 @@
+:::note 
```

Comment:
"Admonitions", as Docusaurus calls them, need empty lines around them. So I think it should be:

:::note

[content]

:::

https://docusaurus.io/docs/markdown-features/admonitions

Reply:
Updated. Thank you for your detailed notes!